bin/node-template/runtime/Cargo.toml

```diff
@@ -23,13 +23,12 @@ system = { version = "2.0.0-rc5", default-features = false, package = "frame-sys
 timestamp = { version = "2.0.0-rc5", default-features = false, package = "pallet-timestamp", path = "../../../frame/timestamp" }
 transaction-payment = { version = "2.0.0-rc5", default-features = false, package = "pallet-transaction-payment", path = "../../../frame/transaction-payment" }
 frame-executive = { version = "2.0.0-rc5", default-features = false, path = "../../../frame/executive" }
-serde = { version = "1.0.101", optional = true, features = ["derive"] }
+serde = { version = "1.0.101", default-features = false, optional = true, features = ["derive"] }
```

Comment:
we don't use serde in no_std so actually this is not needed, but anyway its fine,